bindings/protocols/bacnet/index.html

```diff
@@ -0,0 +1,727 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset='utf-8'>
+    <script src='https://www.w3.org/Tools/respec/respec-w3c' class='remove'></script>
+    <script class='remove'>
+        var respecConfig = {
+            specStatus: "ED",
+            group: "wg/wot",
+            editors: [{
+                name: "Klaus Hartke",
+                w3cid: "107859",
+                company: "Siemens AG",
+                companyURL: "https://www.siemens.com/"
+            },
+            {
+                name: "Michael Koster",
+                w3cid: "110658",
+                company: "Invited Expert",
+                companyURL: "https://github.com/mjkoster"
+            }],
```

Comment:
Now I have a W3C id:

```suggestion
            },
            {
                name: "Dogan Fennibay",
                w3cid: "147431",
                company: "Siemens AG",
                companyURL: "https://www.siemens.com/"
            }],
```